File: Forasoft/JavaVersion/app/src/main/java/com/pustovit/forajava/repository/model/AlbumResponseData.java
package com.pustovit.forajava.repository.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * This class is the response to a request from the service.
 *
 * Created by Pustovit Vladimir on 26.01.2020.
 * vovapust1989@gmail.com
 */

public class AlbumResponseData {

    @SerializedName("resultCount")
    @Expose
    private Integer albumCount;
    @SerializedName("results")
    @Expose
    private List<Album> albums = null;


    public AlbumResponseData() {
    }

    public Integer getAlbumCount() {
        return albumCount;
    }

    public void setAlbumCount(Integer resultCount) {
        this.albumCount = resultCount;
    }

    public List<Album> getAlbums() {
        return albums;
    }

    public void setAlbums(List<Album> results) {
        this.albums = results;
    }


}

File: Forasoft/KotlinVersion/app/src/main/java/com/pustovit/kotlinversion/repository/model/AlbumResponseData.kt
package com.pustovit.kotlinversion.repository.model

import com.google.gson.annotations.SerializedName

/**
 * This class is the response to a request from the service.
 *
 * Created by Pustovit Vladimir on 29.01.2020.
 * vovapust1989@gmail.com
 */

data class AlbumResponseData(
    @SerializedName("resultCount") val resultCount: Int,
    @SerializedName("results")  val albums: List<Album>
)


